index.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const { getAllMovies, getPartialMatch, noInput, addNewMovie } = require('./controllers/movies')
+
+const app = express()
+
+app.get('/movies', getAllMovies)
+
+app.get('/movies/:input', getPartialMatch)
+
+app.post('/', bodyParser.json(), addNewMovie)
```

Comment:
Read the requirements more carefully, this should be `/movies`

![image](https://user-images.githubusercontent.com/5775716/79990878-97294a00-847f-11ea-9cdf-b0d0a9e6a17f.png)